doc/features/vectors/README.md

```diff
@@ -2,9 +2,9 @@
 
 ## Native CQL `vector` type
 
-Introduced in Cassandra 5.0, DSE 6.9 and Datastax Astra, a `vector` is represented as a [CqlVector&lt;T&gt;][cqlvector-api].
+Introduced in Cassandra 5.0 and Datastax Astra, a `vector` is represented as a [CqlVector&lt;T&gt;][cqlvector-api].
```

Comment:
I don't know why we'd keep 'Datastax Astra' here?

Reply:
@sylwiaszunejko , i don't think we need to change anything in vectors doc, probably add scylla there when scylla get this feature released.